test/test_helper.rb

```diff
@@ -1,4 +1,7 @@
 ENV["RAILS_ENV"] ||= "test"
+require 'simplecov'
+SimpleCov.start 'rails'
+
```

Comment:
When we get back up to 90% let's revisit this and add [minimum coverage](https://github.com/colszowka/simplecov#minimum-coverage)

```ruby
SimpleCov.minimum_coverage 90
```

There are other options too if we want to get more fancy 🍸

Reply:
Agreed!